miso-dto/src/main/java/uk/ac/bbsrc/tgac/miso/dto/Dtos.java

```diff
@@ -2313,7 +2313,7 @@ public static LibraryTemplateDto asDto(LibraryTemplate from) {
 
     dto.setId(from.getId());
     dto.setAlias(from.getAlias());
-    dto.setProjectId(from.getProject().getId());
+    dto.setProjectIds(from.getProjects().stream().map(project -> project.getId()).collect(Collectors.toList()));
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace this lambda with a method reference. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1612)

Reply:
Yes, please.